deployments/networks/rialto-parachain.yml

```diff
@@ -9,68 +9,74 @@ services:
     entrypoint: >
       /home/user/rialto-parachain-collator
       --alice
+      --node-key=80cf382988364291a7968ae7825c01f68c50d679796a8983237d07fe0ccf363b
```

Comment:
There were some recent changes in networking && I guess our parachain nodes can't connect to each other because of that => I've added stuff similar to what we have in Rialto and Millau